lib/scrabble_player.rb

```diff
@@ -0,0 +1,62 @@
+require_relative '../lib/scrabble'
+require_relative '../lib/scrabble_tilebag'
+
+
+module Scrabble
+
+  class Player
+    attr_accessor :words_played, :score_tracker, :player_tiles, :tile_bag
+    attr_reader :name
+
+    def initialize(name)
+      @name = name.to_s
+      @words_played = [""]
+      @score_tracker = [0]
+      @player_tiles = []
+    end
+
+    def plays
+      return @words_played
+    end
+
+    def won?
+      @score_tracker.inject { |sum, n| sum + n } > 100 ? true : false #determines winner based on number of points
+    end
+
+
+    def play(word)
+
+      raise ArgumentError.new "Error. You cannot play an empty word. Please try again." if word =~ /^$|\s+/
+
+      raise ArgumentError.new "Error. You cannot play anymore. Please restart game." if @score_tracker.inject { |sum, n| sum + n } > 100
```

Comment:
Couldn't you simply call `won?` here?

Also it **Should** be returning false instead of raising an `ArgumentError`.  Check the instructions